### DISCUSSION TITLE
ReactCompilerConfig API Documentation

### DISCUSSION THREAD

**Discussion by joemun** [reactwg/react-compiler#13] [General Questions]
The official doc site (https://react.dev/learn/react-compiler) appears to be missing the available configuration properties for `ReactCompilerConfig`.  Is there a doc page somewhere, without needing to dig into the source code?

BTW stoked to see that it supports `sources` config now! (instead of using Babel's overrides/test)

Thanks!

**Comment by joemun** [comment]
Found the source here: https://github.com/facebook/react/blob/main/compiler/packages/babel-plugin-react-compiler/src/Entrypoint/Options.ts#L188

Now, wondering about guidance on how to set `runtimeModule`. Looks like this is the successor to the `enableUseMemoCachePolyfill` property?

**Comment by poteto** [comment]
Yeah we should definitely document some of these but there are a lot of options that aren't meant to be used directly.

As for `runtimeModule`, yeah that's similar to `enableUseMemoCachePolyfill` but takes a string with the name of the module that exports the cache function. For React Native apps that aren't on R19 yet, you can do something like this: https://gist.github.com/poteto/37c076bf112a07ba39d0e5f0645fec43#file-babel-config-js-L6
